.circleci/config.yml

```diff
@@ -9,23 +9,22 @@ jobs:
           keys:
             - node-cache-{{ checksum "package-lock.json" }}
             - node-cache
-      - run: npm install
-      - run: npx lerna bootstrap
-      - run: npm run build
+      - run: yarn install
+      - run: yarn build
       - run:
           name: prettier
-          command: npm run prettier:check
+          command: yarn prettier:check
       - run:
           name: lint
-          command: npm run eslint -- --format junit -o reports/junit/js-lint-results.xml
+          command: yarn eslint -- --format junit -o reports/junit/js-lint-results.xml
```

Comment:
yarn doesn't need the extra `--` you can just do `yarn eslint --format junit...`

Reply:
👍